spec/support/configuration_helpers.rb

```diff
@@ -1,3 +1,5 @@
+require 'bundler/stub_specification'
```

Comment:
This file references to `Bundler::StubSpecification`, but leaves it as an exercise for the reader to `require` it.

Reply:
Thanks reader Marco for fixing it ;D ;D ;D